docs/usage_guide/examples/index.rst

```diff
@@ -0,0 +1,21 @@
+Modin Usage Examples
+====================
+
+This section shows Modin usage examples in different scenarios like Modin on a local/remote cluster,
```

Comment:
How is this different from [this page](https://modin--3864.org.readthedocs.build/en/3864/getting_started/examples.html#usage-examples)? Should it be merged as a section to that page?

Reply:
Yes, I think it should probably be merged to this page.

Reply:
merged

Reply:
Looks good!